tests/cloudpickle_test.py

```diff
@@ -487,7 +491,7 @@ def func():
         '''
```

Comment:
Out of curiosity, wouldn't it work if we use raw string here, like `child_process_script = r''' ... '''`?

Reply:
Oops, nvm, the problem is the path.